src/pages/lib/auth/auth-events/q/platform/[platform].mdx

```diff
@@ -22,17 +22,298 @@ export const getStaticProps = (context) => {
   };
 };
 
-AWS Cognito Auth Plugin sends important events through Amplify Hub.
+<InlineFilter filters={["js", "react-native"]}>
+
+Tapping into auth events allows you to react to user flows in real time and trigger custom business logic. For example, you may want to capture data, synchronize your app's state, and personalize the user's experience. You can listen to and respond to events across the Auth lifecycle such as sign-in and sign-out. In this guide we will describe how you can listen to these auth events.
+
+Before you begin, you will need:
+
+- An Amplify project with the Auth category configured
+- The Amplify libraries installed and configured
+
+## Expose hub events triggered in response to auth actions
+
+You can use Amplify Hub with its built in Amplify Auth events to subscribe a listener using a publish-subscribe pattern and capture events between different parts of your application. The Amplify Auth category publishes in the `auth` channel when auth events such as `signIn`, `signUp`, and `signOut` happen independent from your app code.
+
+<InlineFilter filters={["js"]}>
```

Comment:
is there a reason there is an inline filter JUST for JS here?

Reply:
the info between JS and RN seems repetitive to me

Reply:
@abdallahshaban557 yeah this is for the hyperlinks inline - once we have the new site we can do relative links but for now we have to list JS and RN links separate b/c of the switcher placement